### DISCUSSION TITLE
Ability to use a video or photo as camera feed

### DISCUSSION THREAD

**Discussion by psnehanshu** [Genymobile/feedback#164] [Genymotion SaaS] [answered]
As the title suggests, it is possible in genymotion desktop, and maybe possible with SaaS using tools like OBS, but it will be good if I am just able to upload a video or photo and use it immediately.

**Comment by jmaneyrol69** [comment, marked as answer]
@pSnehanshu we are working on implementing Media injection to Genymotion SaaS, like with Genymotion Desktop. In the near future, it will be possible to use a photo as camera feed. Then, later on, it will be possible to use a video.
We do not have any ETA, unfortunately, but make sure to check out our blog regularily at https://www.genymotion.com/blog/ to get the latest news.

**Comment by jmaneyrol69** [comment]
The feature has been implemented to Genymotion SaaS since June 2026 - see https://www.genymotion.com/blog/media-injection-widget-saas/
